src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php

```diff
@@ -16,11 +16,11 @@ trait AuditTrait
     /**
      * Returns the primary key value of an entity.
      *
-     * @param object $entity
-     *
-     * @return mixed
```

Comment:
This helps PHP-CS-Fixer not to add `mixed` as return type if it is run using PHP 8.x